wordpress-mobile/WordPress-Android
# App Data Coverage: Add more tracking for sharing management

Parent #16343 

This PR adds more tracking for Sharing management

**Merging Notes**
- Please do not merge, just approve. I have another PR [#16430] that will conflict with events added to tracking enums  and depending on the order of merge, I may need to resolve conflicts. Thanks.
 
**To test:**
- Launch the app and log in
- Navigate to My Site “menu” tab
- Tap the “sharing” menu item
- Tap the “manage” cog (under Sharing buttons)
- Tap the “Sharing buttons” to launch the selection list
- Make a change and tap OK
- ✅ Verify the logs contain: `🔵 Tracked: sharing_buttons_edit_sharing_buttons_changed`
- Tap the “Edit More Buttons”
- Make a change and tap OK
- ✅ Verify the logs contain: `🔵 Tracked: sharing_buttons_edit_more_sharing_buttons_changed`

## Regression Notes
1. Potential unintended areas of impact
None, adding new track events

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
